### PR TITLE
Multilingual UI support for the adLabel "N of N" string

### PR DESCRIPTION
**Pull request by anvalchev**
I guess this would be useful for other users.

### CODE REVIEW

**Comment by anvalchev** [comment]
I signed it!

File: src/ad-ui.js [hunk 1]
```diff
@@ -268,7 +268,7 @@ AdUi.prototype.updateAdUi =
   }
   let podCount = ': ';
   if (totalAds > 1) {
-    podCount = ' (' + adPosition + ' of ' + totalAds + '): ';
+    podCount = ' (' + adPosition + ' ' + this.controller.getSettings().adLabelNofN + ' ' + totalAds + '): ';
```
Comment (shawnbuso) [inline src/ad-ui.js]: Line too long - max length is 80 characters.

**Comment by anvalchev** [comment]
Should be ok now.

File: src/ad-ui.js [hunk 2]
```diff
@@ -268,7 +268,8 @@ AdUi.prototype.updateAdUi =
   }
   let podCount = ': ';
   if (totalAds > 1) {
-    podCount = ' (' + adPosition + ' of ' + totalAds + '): ';
+    podCount = ' (' + adPosition + ' ' +
+    this.controller.getSettings().adLabelNofN + ' ' + totalAds + '): ';
```
Comment (shawnbuso) [inline src/ad-ui.js]: Can you indent this line 4 spaces? That falls in line with our style guide. Then we should be good to go. Thanks!

**Pull request merged**